uc-crux-llvm/src/UCCrux/LLVM/Overrides/Polymorphic.hs

```diff
@@ -0,0 +1,75 @@
+{-
+Module       : UCCrux.LLVM.Overrides.Polymorphic
+Description  : A 'PolymorphicLLVMOverride' is one that's maximally polymorphic
+Copyright    : (c) Galois, Inc 2021
+License      : BSD3
+Maintainer   : Langston Barrett <langston@galois.com>
+Stability    : provisional
+-}
+
+{-# LANGUAGE GADTs #-}
```

Comment:
Does this actually require GADTs?

Reply:
Yeah, `ArchOk` is an equational constraint.